### DISCUSSION TITLE
Instructions Crazyradio 2 **new**

### DISCUSSION THREAD

**Discussion by knmcguire** [bitcraze/discussions#664] [Announcements]
If you got yourself a shiny [Crazyradio 2.0](https://www.bitcraze.io/products/crazyradio-2-0/), please note that it's not an immediate plug-and-play and that you need to flash new firmware on it.

Please follow these instructions: https://www.bitcraze.io/documentation/tutorials/getting-started-with-crazyradio-2-0/

**Comment by talilama** [comment]
Hi, is there anyway to convert custom firmware for the Crazyradio PA to the Crazyradio 2.0 without having to rewrite it completely?

**Reply by krichardsson** [reply]
No, unfortunately not.

**Reply by simondlevy** [reply]
Thanks for the instructions, Kimberly!  By following them I've been able to get my Crazyradio 2.0 talking to my Crazyflie 2.1 on both a Windows and Linux workstation; however, for some reason I can't get the client to detect the radio on my Windows laptop (Acer Predator / Windows 10 / Python 3.7).  It's a mystery, because I used Zadig to install the driver and can see the Crazyradio 2.0 in Device Manager, but no luck getting the client to talk to it (WARNING:cfclient.ui.main:Cannot find a Crazyradio Dongle).  

Any tips?

**Reply by knmcguire** [reply]
Hi @simondlevy! Please start a new discussion thread for this in Q&A since this is an announcement thread. Just to make it more clear for other people and easier for us to support.
